mantle/cmd/kola/qemuexec.go

```diff
@@ -142,21 +139,24 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
-		configv, _, err := v3.Parse(buf)
+		configv, err := conf.Ignition(string(buf)).Render("", false)
```

Comment:
Not sure why we're setting `configv` here and re-assigning after the error check. Can probably just set `config` here. Previously it was done because `v3.Parse` didn't return a pointer but `conf.Ignition` returns a pointer to the `conf.Conf` object already.

Reply:
Thanks. Updated.